multiurl/http.py

```diff
@@ -468,6 +468,20 @@ def wrapped(url, *args, **kwargs):
         tries = 0
         main_url = url
 
+        if isinstance(retry_after, tuple):
+            sleep_min, sleep_max, sleep_incremental_ratio = retry_after
+        else:
```

Comment:
Should we check it's numeric here? so we can raise a more useful exception

Reply:
Done